### PR TITLE
refactor(Text): replace defaultProps property

### PR DESCRIPTION
**Pull request by silvioprog**
Closes UXD-1640.

### CODE REVIEW

File: src/components/Text/Text.tsx [hunk 1]
```diff
@@ -104,35 +104,27 @@ const variants = {
...
-
 export default Text;
 
 export const Strong = styled(Text).attrs<StrongProps>(() => ({
   as: 'strong',
   isBold: true,
-}))``;
-
-Strong.defaultProps = {
   size: TextSizes.inherit,
```
Comment (ajkl2533) [inline src/components/Text/Text.tsx:126]:
```suggestion
  size: props.size ?? TextSizes.inherit,
```

Comment (silvioprog) [reply src/components/Text/Text.tsx:126]: What should I put instead?

Comment (silvioprog) [reply src/components/Text/Text.tsx:126]: What would you recommend I use instead?

Comment (silvioprog) [reply src/components/Text/Text.tsx:126]: image

Comment (ajkl2533) [reply src/components/Text/Text.tsx:126]: oh you are right I missed that size is omitted for the Strong

**Review by ajkl2533 — APPROVED** [review]

**Pull request merged**